frontend/src/views/scrimmaging.js

```diff
@@ -170,20 +174,28 @@ class ScrimmageHistory extends Component {
             <table className="table table-hover table-striped">
               <thead>
                 <tr>
-                  <th>Date</th>
-                  <th>Time</th>
+                  <th>Creation time</th>
                   <th>Status</th>
                   <th>Score</th>
-                  <th>Team</th>
+                  <th>Opponent</th>
                   <th>Ranked</th>
                   <th>Replay</th>
+                  <th></th>
```

Comment:
Might as well delete this one row? See #436 -- seems better to add logs-exposure only once logs are sanitized or made private to admins (https://mit-battlecode.slack.com/archives/C03SJ354E2K/p1672807451869729)

(assuming this is the fake-header for logs column right)